test/forms/CancelDeclarationSpec.scala

```diff
@@ -17,105 +17,110 @@
 package forms
 
 import base.UnitSpec
+import play.api.data.Form
 import play.api.libs.json.{JsObject, JsString}
 
 class CancelDeclarationSpec extends UnitSpec {
 
-  def formData(lrn: String = "lrn", mrn: String = "123456789012345678", description: String = "description", reason: String = "1") =
-    JsObject(
-      Map(
-        "functionalReferenceId" -> JsString(lrn),
-        "mrn" -> JsString(mrn),
-        "statementDescription" -> JsString(description),
-        "changeReason" -> JsString(reason)
-      )
+  private val validMrn = "123456789012345678"
+  def getBoundedForm(lrn: String = "lrn", mrn: String = validMrn, description: String = "description", reason: String = "1") =
+    CancelDeclaration.form.bind(
+      JsObject(
+        Map(
+          "functionalReferenceId" -> JsString(lrn),
```

Comment:
No need to convert to json. Pass directly the value. You can use `.toString` when it hasn't `String` type.